modules/utxo-lib/package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "coverage-report": "nyc report --reporter=lcov",
     "coverage-html": "nyc report --reporter=html",
-    "coverage": "BITGO_UTXO_LIB_TEST_EXPECTED_COUNT=3443 nyc --check-coverage --branches 90 --functions 90 mocha --recursive",
+    "coverage": "BITGO_UTXO_LIB_TEST_EXPECTED_COUNT=3500 nyc --check-coverage --branches 90 --functions 90 mocha --recursive",
```

Comment:
Is the reason for this change just that the bitcoin gold testnet definition is adding 53 more test case runs to the entire test suite?

Reply:
correct